autotst/data/inputoutput.py

```diff
@@ -122,10 +122,10 @@ def save_ts(self):
             return False
         file_path = self.get_ts_file_path()
         logging.info("Saving TS result file {}".format(file_path))
-        with open(file_path, 'w') as resultFile:
-            resultFile.write('rxnLabel = "{0!s}"\n'.format(self.reaction.label))
-            resultFile.write('method = "{0!s}"\n'.format(self.method))
-            resultFile.write("qmData = {0!r}\n".format(self.qmdata))
+        with open(file_path, 'w') as results_file:
+            results_file.write('rxn_label = "{0!s}"\n'.format(self.reaction.label))
```

Comment:
If we've changed the syntax of our TS results file, do we have a "load" method that can read both old and new syntaxes? or plans to convert all your old data?

Reply:
We should probably make it backward compatible, good comment, this will be true for the `.kinetics` files too